views/index.pug

```diff
@@ -20,11 +20,13 @@ html
         .navtab.dropdown
           div Games
           .dropdown-content
-            a.navtab(onClick='changeActivePage("/games/pacman", "pacman", "Pac-Man")') Pac-Man
+            each gameInfo in gamesList
+              a.navtab(onClick='changeActivePage("/games/' + gameInfo.game + '", "' + gameInfo.game + '", "' + gameInfo.name + '")')= gameInfo.name
```

Comment:
I'm pretty sure Pug supports template strings which would make this cleaner - [https://pugjs.org/language/attributes.html](https://pugjs.org/language/attributes.html)

Reply:
Done https://github.com/knative-portability/Kubercade/pull/29/commits/7b663ae7a0a6368284e78a02441cb1ceb4e3548a